client/index.dev.jsx

```diff
@@ -4,22 +4,24 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import store from './src/store/store';
 
-import Reservation from './src/containers/Reservation';
+import Availability from './src/containers/Availability';
 import PriceSummary from './src/containers/PriceSummary';
 
-const reservation = (
+const randomRoom = Math.floor(Math.random() * 100);
+
+const availability = (
   <Provider store={store}>
-    <Reservation />
+    <Availability roomId={randomRoom} />
   </Provider>
```

Comment:
You can nest components like that?? thats awesome

Reply:
Yes, you can nest components. I think we've done it before in the sprint we did together. We we're passing props around through multiple of nested components right?